### DISCUSSION TITLE
Windows11DragAndDropToTaskbarFix  error

### DISCUSSION THREAD

**Discussion by aioros57** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#13] [Q&A]
Hello,error screen shoot
![Desktop Screenshot 2021 10 27 - 15 51 18 63 (2)](https://user-images.githubusercontent.com/90460028/139080725-f36d516d-f60c-4434-9e3a-7ded15ba9ca0.png)

**Comment by HerMajestyDrMona** [comment]
Please install: https://aka.ms/vs/16/release/vc_redist.x64.exe

I thought they're installed in Windows 11 by default. Maybe you removed "Microsoft Visual C++ 2015 - 2019 Redistributable" in "Add or remove programs" before?

**Comment by jCadeola** [comment]
j'aime bien ton programme ca fonction pas mal
